pynest/nest/lib/hl_api_types.py

```diff
@@ -880,3 +883,51 @@ def apply(self, spatial_nc, positions=None):
                 if len(pos) != len(positions[0]):
                     raise ValueError('All positions must have the same number of dimensions')
             return sli_func('Apply', self._datum, {'source': spatial_nc, 'targets': positions})
+
+
+def serializable(data):
+    """Make data serializable for JSON.
+
+    Parameters
+    ----------
+    data : any
+
+    Returns
+    -------
+    data_serialized : str, int, float, list, dict
+        Data can be encoded to JSON
+    """
+
+    if isinstance(data, (numpy.ndarray, NodeCollection)):
```

Comment:
A "return-early" pattern might be preferably here.

Reply:
Which condition for 'return-early' do you have in mind?

Reply:
The pattern is to write:
```Python
if a:
    return x
if b:
   return y
return z
```

instead of 
```Python
if a:
   return x
elif b:
   return y
else:
   return z
```

Reply:
Ok, I see. I do not any arguments against your suggestion.